packages/sdk/docs/handlers.md

```diff
@@ -0,0 +1,325 @@
+
+2. **Safe Existence Check:** It checks if the specified safe exists in ComposeDB. If the safe does not exist, it proceeds to create a new safe entry in ComposeDB, populating it with data fetched from an external API (specified by `API_ENDPOINT(network)`).
+
+3. **Safe Data Composition:** If the safe needs to be created, the function composes a new safe entity in ComposeDB, including all relevant details like the network ID, safe address, version, and other parameters critical to the safe's operation and identification.
+
+4. **Signer Validation and Creation:** The handler ensures that the signer (transaction initiator) exists in ComposeDB. If the signer is not already present, it creates a new signer entity and establishes a relationship between the signer and the safe, indicating the signer's permission to initiate transactions for the safe.
+
+5. **Transaction Creation:** It constructs the transaction entity, including details like the transaction hash, recipient, value, data, operation type, and nonce. This step is pivotal, as it records the transaction's intent and associated parameters in ComposeDB for traceability and auditing purposes.
+
+6. **Transaction Confirmation:** For each new transaction, a confirmation entity is also created, linking the transaction to its signer and including the signer's signature. This mechanism ensures that each transaction is backed by verifiable consent from authorized signers.
+
+7. **Idempotency and Error Handling:** The function checks if a transaction with the specified nonce already exists for the safe. If so, it follows the principle of idempotency by not duplicating the transaction. This behavior aligns with ensuring consistent and reliable transaction management.
+
+### Interactions with ComposeDB
+
+The handler extensively interacts with ComposeDB to store, retrieve, and manage decentralized data related to transactions, safes, signers, and confirmations. By leveraging ComposeDB's capabilities, `handleCreateTransaction` ensures that all transaction-related data is maintained in a decentralized, secure, and auditable manner. These interactions are facilitated through various utility functions such as `checkSafeExists`, `composeSafe`, `checkSignerExists`, `composeSigner`, and more, each serving a specific purpose in the data management lifecycle.
+
+## handleUpdateConfirmations
+
+The `handleUpdateConfirmations` function plays a crucial role within the dSafe SDK, focusing on updating the confirmation status of a transaction within the ComposeDB infrastructure. This handler ensures that transactions executed within the dSafe ecosystem can be confirmed by authorized signers, reflecting the decentralized governance model intrinsic to many blockchain applications. Below is a detailed exploration of its operation and interaction with ComposeDB.
```

Comment:
`crucial role` - too much water, please clean the text, ChatGPT is made to populate text with meaningless words, I would clean the text, before this comment and after it